src/CodeFormatter.cs

```diff
@@ -176,6 +180,7 @@ await DotNetHelper.PerformRestoreAsync(solutionOrProjectPath, logger) != 0)
 
             var documentsCoveredByEditorConfig = ImmutableArray.CreateBuilder<DocumentId>(totalFileCount);
             var documentsNotCoveredByEditorConfig = ImmutableArray.CreateBuilder<DocumentId>(totalFileCount);
+            var sourceGeneratedDocuments = ImmutableArray.CreateBuilder<DocumentId>(16);
```

Comment:
Was 16 chosen for any specific reason? You may also consider using a named argument here.